test/integration/nockFixtures/PlaybackIds/get.json

```diff
@@ -0,0 +1,99 @@
+[
+    {
+        "scope": "https://api.mux.com:443",
+        "method": "POST",
+        "path": "/video/v1/assets",
```

Comment:
this nockFixture looks like it's for the wrong request, try deleting an re-generating

Reply:
oh wait, nvm. I see the test for this does create asset, get playback id, then delete asset

Reply:
Okay good, thought it may be a little unusual but for testing purposes, figured it was okay